dashboard/config/scripts/allthethings.script

```diff
@@ -1,5 +1,6 @@
 hideable_stages true
 stage_extras_available true
+family_name ''
```

Comment:
note: when i modified `allthethings.script` to add a new stage/level, some properties we've updated were pulled in as well (just the addition of `family_name ''` and how we label assessments)